sources/17-069r3/sections/03-references.adoc

```diff
@@ -2,22 +2,66 @@
 
-* [[[openapi,Open API Initiative]]], Open API Initiative: *OpenAPI Specification 3.0.2*, 2018 https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.2.md
+* [[[gmlsf,OGC 10-100r3]]], van den Brink, L., Portele, C., Vretanos, P.: OGC 10-100r3, *Geography Markup Language (GML) Simple Features Profile*, 2012 http://portal.opengeospatial.org/files/?artifact_id=42729
 
-* [[[rfc2616,RFC 2616]]], Fielding, R., Gettys, J., Mogul, J., Frystyk, H., Masinter, L., Leach, P., Berners-Lee, T.: IETF RFC 2616, *HTTP/1.1*, 1999 http://tools.ietf.org/rfc/rfc2616.txt
-
-* [[[rfc2818,RFC 2818]]], Rescorla, E.: IETF RFC 2818, *HTTP Over TLS*, 2000 http://tools.ietf.org/rfc/rfc2818.txt
+* [[[rfc7946,RFC 7946]]], Butler, H., Daly, M., Doyle, A., Gillies, S., Hagen, S., Schaub, T.: IETF RFC 7946, *The GeoJSON Format*, 2016 https://tools.ietf.org/rfc/rfc7946.txt
 
-* [[[rfc3339,RFC 3339]]], Klyne, G., Newman, C.: IETF RFC 3339, *Date and Time on the Internet: Timestamps*, 2002 http://tools.ietf.org/rfc/rfc3339.txt
+* [[[html5,W3C html5]]], W3C: *HTML5*, W3C Recommendation, 2019 http://www.w3.org/TR/html5/
 
-* [[[rfc8288,RFC 8288]]], Nottingham, M.: IETF RFC 8288, *Web Linking*, 2017 http://tools.ietf.org/rfc/rfc8288.txt
+* [[[schema,1]]], *Schema.org:* http://schema.org/docs/schemas.html
 
-* [[[gmlsf,GMLSF]]], van den Brink, L., Portele, C., Vretanos, P.: OGC 10-100r3, *Geography Markup Language (GML) Simple Features Profile*, 2012 http://portal.opengeospatial.org/files/?artifact_id=42729
+* [[[rfc2616,RFC 2616]]], Fielding, R., Gettys, J., Mogul, J., Frystyk, H., Masinter, L., Leach, P., Berners-Lee, T.: IETF RFC 2616, *HTTP/1.1*, 1999 http://tools.ietf.org/rfc/rfc2616.txt
 
-* [[[rfc7946,RFC 7946]]], Butler, H., Daly, M., Doyle, A., Gillies, S., Hagen, S., Schaub, T.: IETF RFC 7946, *The GeoJSON Format*, 2016 https://tools.ietf.org/rfc/rfc7946.txt
+* [[[rfc2818,RFC 2818]]], Rescorla, E.: IETF RFC 2818, *HTTP Over TLS*, 2000 http://tools.ietf.org/rfc/rfc2818.txt
 
-* [[[html5, HTML5]]], W3C: *HTML5*, W3C Recommendation, 2019 http://www.w3.org/TR/html5/
+[%bibitem]
+=== Date and Time on the Internet: Timestamps
+id:: rfc3339
+docid::
+docid.type:: IETF
+docid.id:: RFC 3339
+contributor::
+contributor.role:: author
+contributor.person.name.Initial:: G.
+contributor.person.name.surname:: Klyne
+contributor::
+contributor.role:: author
+contributor.person.name.Initial:: C.
+contributor.person.name.surname:: Newman
+contributor::
+contributor.organization.name:: IETF
+contributor.organization.abbreviation:: IETF
+contributor.organization.identifier.type:: uri
+contributor.organization.identifier.id:: www.ietf.org
+contributor.role:: publisher
+link::
+type::: src
+content::: http://tools.ietf.org/rfc/rfc3339.txt
+date::
+type::: published
+value::: 2002
 
-* [[[schema,Schema.org]]], *Schema.org:* http://schema.org/docs/schemas.html
\ No newline at end of file
+[%bibitem]
```

Comment:
Again this should be auto-fetch

Reply:
Same case as above (https://github.com/metanorma/mn-samples-ogc/issues/67#issuecomment-606785420).
Do you still want me to apply this change?

Reply:
As above

Reply:
Done.